### DISCUSSION TITLE
Discord guilds channel change suggestion

### DISCUSSION THREAD

**Discussion by Leon-02** [chromiecraft/chromiecraft#3520] [Suggestions]
I've checked out the discord guilds channel and i see that several people post/spam their guild(s) message every day, every couple of days, etc.
now my problem with this, is that i don't want to spend 30 minutes browsing through the same 4 or 5 guild messages.
My suggestion: Make it so that people can only post ONE message for their guild that they can edit whenever they want, that way people can look at every guild easily and without wasting time scrolling through the same messages over and over.

**Comment by 55Honey** [comment]
Hi Leon-02,
Thanks for your suggestion.

The daily re-posting of guild recruiting does actually serve a purpose. If we had a list of guilds with updated posts, somebody would need to maintain them. Dead guilds would need to be removed regularly.

The current system removes that requirement since all active guilds can keep posting.

**Reply by Leon-02** [reply]
I understand i actually didn't think about that, my bad thanks for replying
